src/sections/product-main-template.liquid

```diff
@@ -232,20 +232,20 @@
                   <p class="affirm-as-low-as" data-amount="{{current_variant.price}}"></p>
                 </div>
               {% endif %}
-      {% endunless %}
+          {% endunless %}
               <div class="grid">
                   <div class="grid__item">
                     <div class="margin-15">
                       <h6 class="inline">{{'products.product.warranty' | t | capitalize}}:</h6> 
                       <p class="inline">{{ product.metafields["global"]["Warranty"] }}</p>
                     </div>
                   </div>
-        {% unless noquantity %}
+              {% unless noquantity %}
                   <div class="grid__item small--one-whole medium--one-half large--one-half">
                     <h6>{{'products.product.quantity' | t | capitalize}}:</h6>
                     <input type="number" id="QuantitySmall" value="1" min="1" onchange="bt_product.changeQuantity(this)">
                   </div>
-        {% endunless %}
+              {% endunless %}
                 {% unless nocolor %}
```

Comment:
should this be indented under the endunless above it?

Reply:
this might be a github issue let me pull the branch

Reply:
Take another pass on fixing the indentation on the product-main-template.liquid it's not quite right.